index.js

```diff
@@ -477,7 +370,7 @@ function studentDebtCalculator( financials ) {
   // Institutional Loan debt at graduation
   data.institutionalLoanTotal = calcDebt( data.institutionalLoan,
     data.institutionalLoanRate, data.programLength, data.deferPeriod );
-      
+
   // Home Equity Loans at graduation
   data.homeEquityTotal = (data.homeEquity * .079 / 12 * ((data.programLength * (data.programLength + 1) / 2 * 12)));
```

Comment:
Fixed in https://github.com/mistergone/student-debt-calc/commit/aeb0a24d12734c9d73229bb3ebf0aa2bbd761f28! :fireworks: :octopus: